Dockerfile

```diff
@@ -5,29 +5,44 @@ RUN apt-get update
 RUN apt-get --no-install-recommends install --yes \
     libaio1 libaio-dev xmlsec1 libffi-dev \
     libldap2-dev libsasl2-dev \
-    build-essential libmysqlclient-dev git
+    build-essential default-libmysqlclient-dev git
 
 WORKDIR /tmp/
 
-COPY student_explorer/dependencies/oracle-instantclient12.1-*.deb /tmp/
-RUN dpkg -i oracle-instantclient12.1-*.deb
-ENV ORACLE_HOME /usr/lib/oracle/12.1/client64
-ENV LD_LIBRARY_PATH /usr/lib/oracle/12.1/client64/lib
-RUN rm oracle-instantclient12.1-*.deb
+COPY requirements.txt /tmp/
 
-RUN pip install cx_Oracle==5.3
-RUN pip install gunicorn
+RUN pip install -r requirements.txt && \
+    pip install gunicorn
 
-COPY requirements.txt /tmp/
-RUN pip install -r requirements.txt
+#https://github.com/jwilder/dockerize
+ENV DOCKERIZE_VERSION v0.6.1
+RUN curl -sLO "https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz" \
+    && tar -C /usr/local/bin -xzvf "dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz" \
+    && rm "dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz"
 
-RUN mkdir -p /usr/src/app
+# Sets the local timezone of the docker image
+ENV TZ=America/Detroit
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 
+RUN mkdir -p /usr/src/app
 
 WORKDIR /usr/src/app
 
+COPY . /usr/src/app
+
+ARG LOCALHOST_DEV
+# Run these only for dev
+# Make a python package:
+RUN if [ "$LOCALHOST_DEV" ] ; then \
+    echo "LOCALHOST_DEV is set, building development dependencies" && \ 
+    touch /usr/src/app/student_explorer/local/__init__.py && \
```

Comment:
This command (touch) creates this file on localhost. This file has special script in it that should only be available in localhost instance. We could commit it and copy it instead of creating it. I just copied this pattern from the Vagrantfile.